src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+    private int firstSide;
+    private int secondSide;
+    private int thirdSide;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure(String color, int firstSide) {
+        this.color = color;
+        this.firstSide = firstSide;
+    }
+
+    public Figure(String color, int firstSide, int secondSide) {
+        this.color = color;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public Figure(String color, int firstSide, int secondSide, int thirdSide) {
+        this.color = color;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.thirdSide = thirdSide;
+    }
```

Comment:
Why do you need so many constructors?

Reply:
i thought it was more comfortable then create new ints in every class

Reply:
i thought it will be better than to create new ints in every class